### PR TITLE
feat: add fieldttl command that returns the ttl of the member

### PR DESCRIPTION
**Pull request by romange**
Currently implemented only for saddex.

### CODE REVIEW

File: src/server/generic_family.cc [hunk 1]
```diff
@@ -1495,6 +1536,7 @@ void GenericFamily::Register(CommandRegistry* registry) {
       << CI{"SCAN", CO::READONLY | CO::FAST | CO::LOADING, -2, 0, 0, 0, acl::kScan}.HFUNC(Scan)
       << CI{"TTL", CO::READONLY | CO::FAST, 2, 1, 1, 1, acl::kTTL}.HFUNC(Ttl)
       << CI{"PTTL", CO::READONLY | CO::FAST, 2, 1, 1, 1, acl::kPTTL}.HFUNC(Pttl)
+      << CI{"FIELDTTL", CO::READONLY | CO::FAST, 3, 1, 1, 1, acl::kFieldTtl}.HFUNC(FieldTtl)
```
Comment (dranikpg) [inline src/server/generic_family.cc:1539]: Maybe SetTTL would be a better name? Field doesn't indicate what type it really belongs to

File: src/server/generic_family.cc [hunk 2]
```diff
@@ -618,6 +619,27 @@ OpStatus OpExpire(const OpArgs& op_args, string_view key, const DbSlice::ExpireP
...
+// returns -2 if the key was not found, -3 if the field was not found,
+// -1 if ttl on the field was not found.
+OpResult<long> OpFieldTtl(Transaction* t, EngineShard* shard, string_view key, string_view field) {
+  auto& db_slice = shard->db_slice();
+  const DbContext& db_cntx = t->GetDbContext();
+  auto [it, expire_it] = db_slice.FindExt(db_cntx, key);
+  if (!IsValid(it))
+    return -2;
+
+  if (it->second.ObjType() != OBJ_SET)  // TODO: to finish for hashes.
```
Comment (dranikpg) [inline src/server/generic_family.cc:631]: But then we also want to have one command for both set and hset... maybe we can just add two 😄 Either way it your call

**Review by dranikpg — DISMISSED** [review]

File: src/server/generic_family.cc (hunk 1 above)
Comment (romange) [reply src/server/generic_family.cc:1539]: that's the whole point. I am going to add transparent support for hash family via the same command

File: src/server/generic_family.cc (hunk 2 above)
Comment (romange) [reply src/server/generic_family.cc:631]: the feature is not yet finished, hence the TODO :)

**Review by dranikpg — APPROVED** [review]

**Pull request merged**